src/main/java/com/expediagroup/hiveberg/IcebergInputFormat.java

```diff
@@ -61,7 +61,7 @@
   private static final Logger LOG = LoggerFactory.getLogger(IcebergInputFormat.class);
 
   static final String TABLE_LOCATION = "location";
-  static final String TABLE_FILTER_SERIALIZED = "iceberg.filter.serialized";
+  static final String TABLE_FILTER_SERIALIZED = "hive.io.filter.expr.serialized";
```

Comment:
Ah yes, I've added that in :))

Reply:
Same comment on the other PR, can we just use the variable directly from Hive instead of redeclaring it? Although I guess this is preferable for now until we decide how tied into Hive we want to make this InputFormat.